### PR TITLE
Extract pad-to from issue/diff app

### PR DESCRIPTION
**Pull request by MeBrei**

### CODE REVIEW

File: rad/prelude/strings.rad [hunk 1]
```diff
@@ -107,6 +108,28 @@
   [ (lines " \n \t") ==> [" " " \t"]]
   [ (lines "") ==> []])
 
+;; pad-to
+
+(def pad-to
+  "Appends the `word` with whitespace to get to length `l`. If `word` is longer
```
Comment (jameshaydon) [inline rad/prelude/strings.rad]: Don't most "padding" function just return the input rather than truncate?

Comment (MeBrei) [reply rad/prelude/strings.rad]: You are right. We need the truncate-or-pad functionality to format the list tables. Would `pad-or-truncate-to` work, or can you think of a shorter function name?

Comment (jameshaydon) [reply rad/prelude/strings.rad]: I would probably implement `pad` and then just use `(take _ (pad _))` at the call site. If it is used many times maybe call it `pad-truncate`.

Comment (MeBrei) [reply rad/prelude/strings.rad]: I like the `(take _ (pad _))` option. It is used only twice so far. A more accurate naming would be `pad-right-to` probably.

**Pull request merged**